### PR TITLE
Validate contact form

### PR DESCRIPTION
**Pull request by chepkok3**
Implemented the following interactions through JS form validation in the portfolio:
-When the user submits the form, you check if the email is in lowercase.
-If the validation is OK, the form is sent.
-If the validation is not OK, you show an error message to the user near the submit button informing them of the error, and 
 the form is not sent.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -173,7 +173,8 @@ <h2 class="contact-title">Contact me</h2>
             action="https://formspree.io/f/xpznvyer"
             method="POST"
             class="contact-form"
-            
+            id="contact-me-form"
+            name="contactMe"
```
Comment (codepantha) [inline index.html:177]: - [ ]  I noticed that the input fields in the contact form do not allow me to enter any information. Please try to fix this issue in order to get approved ✔️

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors
✔️ Collaborative effort

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please consider creating a separate LICENSE file and link to it from your README. It is certainly better than having your license in the README file as it clutters your README with information which does not belong there 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (chepkok3) [reply index.html:177]: @codepantha Okay

**Review by AnselemOdims — APPROVED** [review]
### Hi  @chepkok3 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**